swift/ql/lib/codeql/swift/controlflow/internal/ControlFlowGraphImpl.qll

```diff
@@ -1444,6 +1444,24 @@ module Exprs {
     }
   }
 
+  /** Control-flow for Move Semantics. See the QLDoc for `CopyExpr` for details. */
```

Comment:
Are these any details to be found on the QLDoc for `CopyExpr`? Since these classes are all private you don't _have_ to provide QLDoc to these classes if there's nothing to explain. I'd much rather have no comment than a false comment 😂